lib/get-youtube-url.js

```diff
@@ -3,14 +3,17 @@ const qs = require('sdk/querystring');
 
 module.exports = getYouTubeUrl;
 
-function getYouTubeUrl(videoId, cb, time) {
+function getYouTubeUrl(videoId, cb, time, cc) {
+  if (!cc) cc = false;
+
   const params = qs.stringify({
     autoplay: time ? 1 : 0,
     modestbranding: 1,
     controls: 0,
     disablekb: 0,
     enablejsapi: 1,
     fs: 0,
+    cc_load_policy: +cc, // force boolean to number
```

Comment:
Seems weird to use `Boolean(foo)` instead of `!!foo`, but then use `+foo` instead of `Number(foo)`.

I personally prefer the shorthand version, but I care more that it's consistent. Your call on what approach to pick.